.github/workflows/pre-commit.yml

```diff
@@ -32,26 +32,26 @@ jobs:
         directory: ${{ fromJson(needs.collectInputs.outputs.directories) }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Terraform min/max versions
         id: minMax
-        uses: clowdhaus/terraform-min-max@v1.0.3
+        uses: clowdhaus/terraform-min-max@v1.2.0
         with:
           directory: ${{ matrix.directory }}
 
       - name: Pre-commit Terraform ${{ steps.minMax.outputs.minVersion }}
         # Run only validate pre-commit check on min version supported
-        if: ${{ matrix.directory !=  '.' }}
-        uses: clowdhaus/terraform-composite-actions/pre-commit@v1.3.0
+        if: ${{ matrix.directory !=  '.' && !contains(matrix.directory, 'wrappers') }}
```

Comment:
The correct way is probably to disable `clowdhaus/terraform-min-max` from fetching terraform versions from the `wrappers` directories, but the simplest is to disable runs of `pre-commit terraform_validate` step, so I did.

Wrappers define just the minimum required versions of the Terraform for itself, but they don't know (and should not know) what they are wrapping and what is their required version of the Terraform. Hope it makes sense :)

What do you think?

Reply:
this works!